tests/e2e/pageobjects/ide/Editor.ts

```diff
@@ -332,19 +332,33 @@ export class Editor {
         await this.driverHelper.waitDisappearanceWithTimeout(errorInLineLocator, timeout);
     }
 
+    async waitWarningInLine(lineNumber: number, timeout: number = TimeoutConstants.TS_ERROR_HIGHLIGHTING_TIMEOUT) {
+        Logger.debug(`Editor.waitWarningInLine line: "${lineNumber}"`);
+
+        const warningInLineLocator: By = await this.getWarningInLineLocator(lineNumber);
+        await this.driverHelper.waitVisibility(warningInLineLocator, timeout);
+    }
+
+    async waitWarningInLineDisappearance(lineNumber: number, timeout: number = TimeoutConstants.TS_ERROR_HIGHLIGHTING_TIMEOUT) {
```

Comment:
Does it mean "InLineWarning" ?

Reply:
https://github.com/eclipse/che/pull/19205#discussion_r587369568